hpke.go

```diff
@@ -214,10 +215,10 @@ func newCipherContext(suite CipherSuite, setupParams setupParameters, contextPar
 		return cipherContext{}, err
 	}
 
-	return cipherContext{key, nonce, exporterSecrert, aead, 0, suite.KDF, nil, setupParams, contextParams}, nil
+	return cipherContext{key, nonce, exporterSecrert, aead, 0, suite, nil, setupParams, contextParams}, nil
```

Comment:
This "secrert" typo has been bugging me :)

Reply:
Hah! Nice. I'll fix it.